config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml

```diff
@@ -319,6 +325,7 @@ periodics:
     preset-azure-cred-only: "true"
     preset-windows-private-registry-cred: "true"
     preset-capz-windows-common-main: "true"
+    preset-capz-gmsa-setup: "true"
```

Comment:
I don't think we run the serial slow jobs on containerd nightly?

Reply:
you are correct

Reply:
updated